prover/protocol/compiler/globalcs/evaluation.go

```diff
@@ -37,7 +37,10 @@ type evaluationProver evaluationCtx
 
 // evaluationVerifier wraps [evaluationCtx] to implement the [wizard.VerifierAction]
 // interface.
-type evaluationVerifier evaluationCtx
+type evaluationVerifier struct {
+	evaluationCtx
+	skipped bool
```

Comment:
We set this field to true for the verifier steps that are done by the ful-recursion circuit